.github/workflows/end_to_end_tests.yaml

```diff
@@ -40,21 +50,22 @@ jobs:
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: '8.4'
-                  extensions: none, ctype, iconv, intl, mbstring, pdo_sqlite, xml
-                  tools: composer
                   coverage: none
+                  extensions: none, ctype, dom, iconv, intl, mbstring, pdo_sqlite, simplexml, tokenizer, xml, xmlwriter
+                  php-version: latest
```

Comment:
Since the above test with `symfony new` also does use `composer create-project` internally I opted for a different PHP version for this job